SU-SWS/stanford_profile_helper
# D8CORE-2950: adding mailto icon to wysiwyg

# READY FOR advice


# Summary
This adds the mailto icon to the wysiwyg

# Needed By (Date)

# Urgency
- How critical is this PR?
not

# Steps to Test

1. pull this branch
1. Clear the cache
1. Create mailto links, or copy from https://amptesting.sites.stanford.edu/interior-page/wysiwyg/icons-links
1. In the editor you should see the email icons

Optionally if images aren't showing:
1. navigate to /stanford_profile_styles/node_modules
1. Delete the decanter directory (rm -rf decanter)
1. Clone decanter into /node_modules
1. Checkout this branch of Decanter: https://github.com/SU-SWS/decanter/pull/776
1. Build Decanter (npm run build)
1. Build this (npm run publish) to bring in the new images.


# Affected Projects or Products
- D8Core

# Associated Issues and/or People
- JIRA ticket
https://stanfordits.atlassian.net/browse/D8CORE-2950
https://stanfordits.atlassian.net/browse/D8CORE-3169
- Other PRs
https://github.com/SU-SWS/decanter/pull/816
https://github.com/SU-SWS/decanter/pull/776
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
See issue:
https://github.com/SU-SWS/decanter/issues/621
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
